articles/140_topic_and_service_name_mapping.md

```diff
@@ -305,6 +314,24 @@ In ROS 2, topic and service names differ from ROS 1 in that they:
 
 This section lists concerns about the proposed design and alternatives that were considered.
 
+#### Substitution of the Namespace Delimiter
```

Comment:
Also, this needs a section preceding it to describe the context in which it was rejected. Just moving it here is insufficient in my opinion...